modules/cram/templates/stragler_call.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euo pipefail
+
+convert_to_bam () {
+  ${CMD_SAMTOOLS} view --reference "!{paramReference}" --bam --output "!{cram}.bam" --threads "!{task.cpus}" "!{cram}"
+  ${CMD_SAMTOOLS} index "!{cram}.bam"
+}
+
+convert_to_bam_cleanup () {
+  rm "!{cram}.bam" "!{cram}.bam.bai"
+}
+
+call_short_tandem_repeats () {
+    local args=()
+    args+=("--loci" "!{paramLoci}")
+    args+=("--sample" "!{sampleId}")
+    args+=("-v" "straglr.vcf")
+    if [ -z "!{sampleSex}" ]; then
+        args+=("--sex" "!{sampleSex}")
+    fi
+    args+=("--min_support" "!{paramMinSupport}")
+    args+=("--min_cluster_size" "!{paramMinClusterSize}")
+    args+=("!{cram}.bam")
+    args+=("!{paramReference}")
+
+    ${CMD_STRAGLR} "${args[@]}"
+}
+
+index () {
+  ${CMD_BGZIP} -c straglr.vcf > !{vcfOut}
```

Comment:
how about using CMD_BCFTOOLS view here to remove the dependency on CMD_BGZIP? this has the additional advantages of validating the output vcf and when updating to BCFtools 1.18 it allows to create the index in the same command.